src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -940,6 +941,13 @@ def _configure_security_policy_storage_params(arg_ctx):
     with self.argument_context('sql db threat-policy update') as c:
         _configure_security_policy_storage_params(c)
 
+        c.argument('security_alert_policy_name',
+                   arg_group=policy_arg_group,
+                   options_list=['--security-alert-policy-name', '-p'],
+                   required=True,
+                   help='The name of the security alert policy. The value is Default.',
+                   arg_type=get_enum_type(SecurityAlertPolicyName))
```

Comment:
Does this parameter have only one allowed value `default`? Can it be added inside cli custom function so that cli customer won't face any breaking change?

Reply:
yes. the `default` only `security_alert_policy_name` is the new added parameter in track2 SDK. 
previously, in track1 SDK, if the enum has only one value, it was assigned as `default` in SDK itself. do you think it is a bug of Python SDK? 

In track2 SDK, the update method interface has changed to https://github.com/Azure/azure-sdk-for-python/blob/ede50054a5b78e1a69785221d7da19a0bb817278/sdk/sql/azure-mgmt-sql/azure/mgmt/sql/operations/_database_security_alert_policies_operations.py#L121

with the `custom_update_command` wrap, the customized func can only change the `parameter`, not `security_alert_policy_name`. Thus, I have to make breaking change for
> sql db threat-policy update command: add --security-alert-policy-name default as required parameter

Reply:
Replied https://github.com/Azure/azure-cli/pull/18637#discussion_r674430846

Reply:
addressed the comment